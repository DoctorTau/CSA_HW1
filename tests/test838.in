125
1409 7849 2274 3822 -6299 3290 -1687 2646 -6437 2144 -3617 8416 446 -7227 -4409 9893 7898 1346 5831 8440 4868 3291 2787 -419 -622 4714 238 64 1503 5696 9959 -751 714 -681 6604 8151 4294 5625 -241 -7467 -7142 9978 5216 8777 -4379 6470 260 -4018 8829 -9050 -9623 9513 -8393 -9593 1421 -8564 -8882 9034 -7398 1016 2168 5515 9594 -7485 6620 8575 2162 -7731 -2887 -6318 6637 -5912 -4860 6846 -1111 199 -3172 7014 5014 -5714 -6279 -2792 -1392 -9451 4682 -7354 5962 3168 161 4933 2364 -2494 1964 8508 89 -5298 -4312 5273 2999 9144 1656 3669 -9760 1929 -3492 2403 -7070 5368 -8227 -4638 3767 -8184 4923 -1087 -9418 7238 372 -3851 -6320 1481 144 7767 8532 1741 8978 