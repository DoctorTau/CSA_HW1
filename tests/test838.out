-622 -7142 -4018 -9050 -8564 -8882 -7398 -6318 -5912 -4860 -3172 -5714 -2792 -1392 -7354 -2494 -5298 -4312 -9760 -3492 -7070 -4638 -8184 -9418 -6320 